server/start.go

```diff
@@ -272,7 +273,14 @@ func startNode(
 
 	nodeKey := &key.NodeKey{PrivKey: signingKey, PubKey: signingKey.GetPublic()}
 
-	rollkitcfg, err := config.Load(rootCmd)
+	cmd := &cobra.Command{}
```

Comment:
I went this way, Load would work if it was called on `start` directly.
I think for other protocols loading directly the flags from the command is what makes sense. In the SDK we have another pre-processing flow and create dynamically the command.
I was thinking, maybe adding a `LoadFromViper` in rollkit as well, but I think it doesn't really makes sense either.